lib/web_ui/lib/src/engine/semantics/accessibility.dart

```diff
@@ -52,9 +52,10 @@ class AccessibilityAnnouncements {
   html.HtmlElement get _domElement => _element ??= _createElement();
 
   /// Decodes the message coming from the 'flutter/accessibility' channel.
-  void handleMessage(ByteData data) {
+  StandardMessageCodec handleMessage(ByteData data) {
```

Comment:
I'd prefer to keep this method `void` and have it accept a message codec as an argument.

Reply:
Done.